app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -109,8 +114,21 @@ private void inflateHomeView(WebView view) {
     home = inflater.inflate(R.layout.content_main, view, false);
     callback.setHomePage(home);
     if (sharedPreferenceUtil.nightMode()) {
+      ImageView sideImage = home.findViewById(R.id.content_side_image);
       ImageView cardImage = home.findViewById(R.id.content_main_card_image);
-      cardImage.setImageResource(R.drawable.ic_home_kiwix_banner_night);
+      AppCompatButton downloadButton = home.findViewById(R.id.content_main_card_download_button);
+      ConstraintLayout constraintLayout = home.findViewById(R.id.constraint_main);
+      constraintLayout.setBackgroundResource(R.drawable.back_cover_night);
+      sideImage.setImageResource(R.drawable.home_side_cover_night);
+      cardImage.setImageResource(R.drawable.kiwix_logo_night);
+      downloadButton.setTextColor(Color.parseColor("#000000"));
+
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        downloadButton.setBackgroundTintList(ColorStateList.valueOf(
+            KiwixApplication.getInstance().getResources().getColor(R.color.complement_blue800)));
+      }
+
+
```

Comment:
Remember to reformat your code before pushing the changes so there are no extra blank lines like this.